### PR TITLE
Update Installer Debugging Documentation

### PR DESCRIPTION
**Pull request by thedoubl3j**
##### SUMMARY

update the installer debugging doc to include specific variable over ride examples.

##### ISSUE TYPE

 - Bug, Docs Fix or other nominal change

##### ADDITIONAL INFORMATION

n/a

### CODE REVIEW

File: docs/debugging.md [hunk 1]
```diff
@@ -35,6 +35,19 @@ ansible_operator_meta:
   namespace: awx
 service_type: nodeport
 ```
+The vars file will replace the awx resource so any value that you wish to over ride using the awx resource, put in the vars file. For example, if you wish to use your own image, version and pull policy, you can specify it like below:
```
Comment (TheRealHaoLiu) [inline docs/debugging.md:38]: override is 1 word

File: docs/debugging.md [hunk 2]
```diff
@@ -35,6 +35,19 @@ ansible_operator_meta:
...
+
+```yaml
+# vars.yml
+---
+ansible_operator_meta:
+  name: awx
+  namespace: awx
+service_type: nodeport
+image: $DEV_DOCKER_TAG_BASE/awx_kube_devel
```
Comment (TheRealHaoLiu) [inline docs/debugging.md:47]: i dont think $DEV_DOCKER_TAG_BASE would resolve here

**Comment by TheRealHaoLiu** [comment]
updated your PR title to be more descriptive

**Comment by shanemcd** [comment]
Shoot. I clicked Merge by mistake. I will follow up with more fixes.

**Pull request merged**